pkg/apis/certmanager/v1alpha2/types_certificate.go

```diff
@@ -72,6 +72,11 @@ const (
 // A valid Certificate requires at least one of a CommonName, DNSName, or
 // URISAN to be valid.
 type CertificateSpec struct {
+	// Full X509 name specification (https://golang.org/pkg/crypto/x509/pkix/#Name).
+	// If specified, overrides any other name elements below.
+	// +optional
+	X509Name *X509DistinguishedName `json:"x509Name,omitempty"`
```

Comment:
Can we rename this field to `Subject`? Looking around, it seems like 'subject' is the standard terminology for this stanza 😄